validator/client/validator.go

```diff
@@ -343,17 +342,21 @@ func (v *validator) UpdateDuties(ctx context.Context, slot uint64) error {
 				if err != nil {
 					return errors.Wrap(err, "could not check if a validator is an aggregator")
 				}
-				if _, err := v.validatorClient.SubscribeCommitteeSubnet(ctx, &ethpb.CommitteeSubnetSubscribeRequest{
-					Slot:         duty.AttesterSlot,
-					CommitteeId:  duty.CommitteeIndex,
-					IsAggregator: aggregator,
-				}); err != nil {
-					return err
-				}
+				subscribeSlots = append(subscribeSlots, duty.AttesterSlot)
+				subscribeCommitteeIDs = append(subscribeCommitteeIDs, duty.CommitteeIndex)
+				subscribeIsAggregator = append(subscribeIsAggregator, aggregator)
 			}
 		}
 	}
 
+	if _, err := v.validatorClient.SubscribeCommitteeSubnets(ctx, &ethpb.CommitteeSubnetsSubscribeRequest{
+		Slots:        subscribeSlots,
+		CommitteeIds: subscribeCommitteeIDs,
+		IsAggregator: subscribeIsAggregator,
+	}); err != nil {
+		return err
+	}
```

Comment:
maybe it can be replaced by
```_, err := v.validatorClient.SubscribeCommitteeSubnets(ctx, &ethpb.CommitteeSubnetsSubscribeRequest{
		Slots:        subscribeSlots,
		CommitteeIds: subscribeCommitteeIDs,
		IsAggregator: subscribeIsAggregator,
	})
return err
}```